include/zephyr/bluetooth/mesh/brg_cfg_cli.h

```diff
@@ -0,0 +1,313 @@
+	 *
+	 *  Handles received Subnet Bridge Status messages from a Bridge
+	 *  Configuration Server.
+
+	 *  @param cli      Bridge Configuration Client context.
+	 *  @param addr     Address of the sender.
+	 *  @param status   Status received from the server.
+	 */
+	void (*subnet_bridge_status)(struct bt_mesh_brg_cfg_cli *cli, uint16_t addr,
+				     uint8_t status);
+
+	/** @brief Optional callback for Bridging Table Size Status message.
+	 *
+	 *  Handles received Bridging Table Size Status messages from a Bridge
+	 *  Configuration Server.
+	 *
+	 *  @param cli      Bridge Configuration Client context.
+	 *  @param addr     Address of the sender.
+	 *  @param size     Size received from the server.
+	 */
+	void (*bridging_table_size_status)(struct bt_mesh_brg_cfg_cli *cli, uint16_t addr,
+					   uint16_t size);
+
+	/** @brief Optional callback for Bridging Table Status message.
+	 *
+	 *  Handles received Bridging Table status messages from a Bridge
+	 *  Configuration Server.
+	 *
+	 *  @param cli      Bridge Configuration Client context.
+	 *  @param addr     Address of the sender.
+	 *  @param rsp      Response received from the Bridging Configuration Server.
+	 */
+	void (*bridging_table_status)(struct bt_mesh_brg_cfg_cli *cli, uint16_t addr,
+				      struct bt_mesh_bridging_table_status *rsp);
+
+	/** @brief Optional callback for Bridged Subnets List message.
+	 *
+	 *  Handles received Bridged Subnets List messages from a Bridge
+	 *  Configuration Server.
+	 *
+	 *  @param cli      Bridge Configuration Client context.
+	 *  @param addr     Address of the sender.
+	 *  @param rsp      Response received from the Bridging Configuration Server.
+	 */
+	void (*bridged_subnets_list)(struct bt_mesh_brg_cfg_cli *cli, uint16_t addr,
+				     struct bt_mesh_bridged_subnets_list *rsp);
+
+	/** @brief Optional callback for Bridging Table List message.
+	 *
+	 *  Handles received Bridging Table List messages from a Bridge
+	 *  Configuration Server.
+	 *
+	 *  @param cli      Bridge Configuration Client context.
+	 *  @param addr     Address of the sender.
+	 *  @param rsp      Response received from the Bridging Configuration Server.
+	 */
+	void (*bridging_table_list)(struct bt_mesh_brg_cfg_cli *cli, uint16_t addr,
+				    struct bt_mesh_bridging_table_list *rsp);
+};
+
+/** Bridge Configuration Client Model Context */
+struct bt_mesh_brg_cfg_cli {
+	/** Bridge Configuration model entry pointer */
+	const struct bt_mesh_model *model;
+
+	/** Event handler callbacks */
+	const struct bt_mesh_bdg_cfg_cli_cb *cb;
+
+	/* Internal parameters for tracking message responses. */
+	struct bt_mesh_msg_ack_ctx ack_ctx;
+};
+
+/** @brief Sends a Subnet Bridge Get message to the given destination address
+ *
+ *  This function sends a Subnet Bridge Get message to the given destination
+ *  address to query the value of the Subnet Bridge state of a subnet. The
+ *  Subnet Bridge state indicates whether the subnet is bridged or not. The
```

Comment:
```suggestion
 *  Subnet Bridge state indicates whether the subnet bridge feature is enabled or not. The
```

and similar fix for `_set()` doc string.